run.py

```diff
@@ -384,9 +384,11 @@ def run_state_level_forecast(min_date, max_date, country="USA", state=None):
 
 if __name__ == "__main__":
     logging.basicConfig(level=logging.INFO)
-    # @TODO: Remove interventions override once support is in the Harvard model.
-    min_date = datetime.datetime(2020, 3, 7)
-    max_date = datetime.datetime(2020, 7, 6)
-    # build_county_summary()
-    run_county_level_forecast(min_date, max_date)
-    run_state_level_forecast(min_date, max_date)
+    with public_data_hash(os.getenv('COVID_DATA_PUBLIC_HASH', None)) as git_hash:
```

Comment:
Can you add this to `run_model.py` as well?

Reply:
Assuming you mean `run_old_model.py`, I don't think this exact approach will work the same way, since without `COVID_DATA_PUBLIC` also being specified, that code will clone a fresh repo and use that. I can look at unifying the two approaches in another PR though.